packages/react-refresh/src/__tests__/ReactFreshBabelPlugin-test.js

```diff
@@ -415,7 +415,7 @@ describe('ReactFreshBabelPlugin', () => {
         }
     `,
         {
-          plugins: ['transform-es2015-modules-commonjs'],
```

Comment:
@gaearon This is the Babel 6 version which was only accidentally available in our repo. This now uses the Babel 7 transform. Not sure what this is testing.

Reply:
I think this just makes sure the transform doesn't break it in some way. Makes sense to use the recent one.